rods_state_to_pcc_conv:
  ros__parameters:
    queue_size: 10
    print_params: 1
    pub_frequency: 2.0
    topic_names: ['/elastica/control_input','/elastica/time_tracker','/elastica/rods_state','/elastica/objs_state',  '/elastica/pcc_kinematic_states', '/elastica/pcc_transformed_poses']
    
forward_kinematics_calc:
  ros__parameters:
    queue_size: 10
    print_params: 1
    pub_frequency: 2.0
    topic_names: ['/elastica/control_input','/elastica/time_tracker','/elastica/rods_state','/elastica/objs_state',  '/elastica/pcc_kinematic_states', '/elastica/pcc_transformed_poses']
    